runtime/vm/OutOfLineINL_java_lang_invoke_NativeMethodHandle.cpp

```diff
@@ -66,6 +66,12 @@ OutOfLineINL_java_lang_invoke_NativeMethodHandle_initJ9NativeCalloutDataRef(J9VM
 	 */
 	U_32 typeCount = J9INDEXABLEOBJECT_SIZE(currentThread, argTypesObject) + 1;
 	args = (ffi_type **)j9mem_allocate_memory(sizeof(ffi_type *) * typeCount, OMRMEM_CATEGORY_VM);
+
+	/* Zero out the memory because if an error occurs before all the entries in this array are initialized then the error
+	 * handling code at freeAllMemoryThenExit will attempt to free all the pointers, some of which will not be initialized.
+	 */
+	memset(args, 0, sizeof(ffi_type *) * typeCount);
```

Comment:
should happen after the null check

Reply:
Right. What a silly mistake. Fixed in 736be5e along with the copyright date.